challenge_03_loops-conditionals/loops-conditionals/loops/exercise_00.js

```diff
@@ -0,0 +1,15 @@
+/*
+    1. Create a for loop that will print out all the even numbers between 10 and 40.
+    
+    2. Create a for loop that will print out all the odd numbers between 10 and 40.
+*/
+for (let evennumbers = 10; evennumbers <= 40; evennumbers++) {
```

Comment:
@Manel0712  @Enma1920 
Pay atention! Variables are defined using camelCase convention
```suggestion
for (let evenNumbers = 10; evenNumbers <= 40; evenNumbers++) {
```

FIX: apply suggestion